source/index.html.md.erb

```diff
@@ -48,6 +48,13 @@ The land method has two parameters, first you need to define the `Rocket Slug` t
 ### Launching a Rocket
 At some point you might want to modify the landed Rocket or even create your own Rocket. You can push your new Rocket to the rocketbase.ai platform by using the launch functionality of the PIP package.
 
+> A rocket can be launched using the `Rocket Slug`
+
+```python
+liftoff = Rocket.launch('username/modelName/<hash>')
+```
+
+To launch a rocket, you can simply use this line in the same Python file as for the previous snippet.
```

Comment:
Hmm yeah. I thought this might be the best way to indicate where this script/snippet should be placed in order for it to find the `rockets` folder. Does that make sense?

Reply:
yes, that's fine. I just wasn't sure whether you wanted to say that.